en: add notices about import/scale with TiUP (#364)
pingcap/docs-dm: en/maintain-dm-using-tiup.md

To use this command, you need to specify at least two arguments: the cluster name and the node ID. The node ID can be obtained by using the `tiup dm display` command in the previous section.

> For example, to scale in the DM-worker node on `172.16.5.140`, run the following command. You can scale in the DM-master using similar methods:

Reviewer: ```suggestion
For example, to scale in the DM-worker node on `172.16.5.140` (similar to scaling in DM-master), run the following command:
```

The scale-out operation has an inner logic similar to that of deployment: the TiUP DM component first ensures the SSH connection of the node, creates the required directories on the target node, then executes the deployment operation, and starts the node service.

> For example, to scale out a DM-worker node in the `prod-cluster` cluster, take the following steps. You can scale in the DM-master using similar methods:

Reviewer: ```suggestion
For example, to scale out a DM-worker node in the `prod-cluster` cluster, take the following steps (scaling out DM-master has similar steps):
```

>
> - TiUP does not support importing the DM Portal component in a DM 1.0 cluster.
> - You need to stop the original cluster before importing. 

> > - TiUP only supports importing to v2.0.0-rc.2 or later versions.

Reviewer: ```suggestion
> - TiUP only supports importing to a DM cluster of v2.0.0-rc.2 or a later version.
```

> - The deployment directories of some components are different from those of the original cluster. You can execute the `display` command to view the details. 
> - Run `tiup update --self && tiup update dm` before importing to make sure that the TiUP DM component is the latest version.

> > - Only one DM-master node is in the cluster after importing. Refer to [Scale out a cluster](#scale-out-a-cluster) to expand the DM-master.

Reviewer: ```suggestion
> - Only one DM-master node exists in the cluster after importing. Refer to [Scale out a cluster](#scale-out-a-cluster) to scale out the DM-master.
```
